index.d.ts

```diff
@@ -339,6 +339,11 @@ export class TimeoutError extends Error {
 	constructor();
 }
 
+/**
+Symbol for aborting retries in the `beforeRetry` hook
+*/
+export const ABANDON_RETRY: Symbol;
```

Comment:
Should be `unique symbol`, see https://github.com/sindresorhus/find-up/blob/6c32f0caed1684ef778053b6f79b13a772e22ba4/index.d.ts#L3 and https://github.com/sindresorhus/find-up/blob/6c32f0caed1684ef778053b6f79b13a772e22ba4/index.d.ts#L134